resources/todomvc/architecture-examples/svelte/src/App.svelte

```diff
@@ -81,40 +85,10 @@
 
         <ul class="todo-list">
             {#each filtered as item, index (item.id)}
-                <li class="{item.completed ? 'completed' : ''} {editing === index ? 'editing' : ''}">
-                    <div class="view">
-                        <input class="toggle" type="checkbox" bind:checked={item.completed} />
-                        <!-- svelte-ignore a11y-label-has-associated-control -->
-                        <label on:dblclick={() => (editing = index)}>{item.description}</label>
-                        <button on:click={() => removeItem(index)} class="destroy" />
-                    </div>
-
-                    {#if editing === index}
-                        <div class="input-container">
-                            <!-- svelte-ignore a11y-autofocus -->
-                            <input value={item.description} data-index={index} id="edit-todo-input" class="edit" on:keydown={handleEdit} on:blur={updateItem} use:focusInput />
-                            <label class="visually-hidden" for="edit-todo-input">Edit Todo Input</label>
-                        </div>
-                    {/if}
-                </li>
+                <Item item={item} editing={editing} index={index} on:removeItem={removeItem} on:updateItem={updateItem} on:startEdit={startEdit} on:handleEdit={handleEdit} on:toggleItem={toggleItem} />
```

Comment:
instead of passing the `index` to the component that doesn't need it (see below), you can use arrow functions as handler that would take the `index` directly here.

Reply:
removed index param